bokehjs/src/less/main.less

```diff
@@ -77,20 +77,32 @@
   display: block;
 }
 
-.bk-tooltip.bk-left::before {
+.bk-tooltip.bk-left.bk-arrow::before {
```

Comment:
@mattpap good catch I agree that is more specific and an improvement. @BoboTiG can you make this last change? I think this might be ready after that.